developers.json

```diff
@@ -0,0 +1,12 @@
+---
+layout: profile
+title: developers
+---
+[{% for item in site.profiles %}
+    {
+        "hiveUserName" : "{{item.hiveUserName}}",
+        "welcomeThreadEntryUrl" : "{{item.welcomeThreadEntryUrl}}",
+        "quote" : "{{item.quote}}"
+    }{%  if forloop.last == false %},{% endif %}{% endfor %}
```

Comment:
can't we improve this line by just checking `!forloop.last`??

Reply:
Let me give it a try.

Reply:
`{%  if !forloop.last %},{% endif %}{% endfor %}`

Like this?

Reply:
yeah, does it work?

Reply:
I tried that one but the comma is missing from the right place.

![image](https://user-images.githubusercontent.com/77311602/129409441-76c5b329-3878-4ce4-a280-db9bf919d0c9.png)

Reply:
I see

Reply:
https://www.geeksforgeeks.org/not-keyword-in-ruby/ 
check this out, apparently you can use `not(<expression>)` to get the same result

Reply:
I tried it as well, but it resulted in a Liquid error because these code lines are written in Liquid template language.

![image](https://user-images.githubusercontent.com/77311602/129413196-09e7c5b6-4bc8-4479-acc6-bbbad2756c83.png)

Reply:
oh then it's alright, let's keep it this way :)

Reply:
Okay sure. Thank you for reviewing this. :)